{{cookiecutter.project_shortname}}/package.json

```diff
@@ -14,7 +14,7 @@
 {%- endif %}
   "main": "build/index.js",
   "scripts": {
-    "start": "webpack-serve --config ./webpack.serve.config.js --open",
+    "start": "webpack serve --config ./webpack.serve.config.js --open",
```

Comment:
Do people still use the demo? I wonder if we'd be better off removing it, so we only explicitly support testing components inside a Dash app.

Reply:
OK, we can leave it. Most of our own components the demo is either nonfunctional or has been deleted entirely, but if others use it we can leave it there.